examples/basic-demo-tableload.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates basic usage of the firefly_client API show_table for the file with multiple tables.\n",
+    "\n",
+    "Note that it may be necessary to wait for some cells (like those displaying an table) to complete before executing later cells."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Imports for Python 2/3 compatibility"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function, division, absolute_import"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Imports for firefly_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from firefly_client import FireflyClient"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we use the irsaviewer application for the server."
```

Comment:
Since we are not using irsaviewer, it would be better to refer to Docker like this:
"In this example we assume the server is running locally, e.g. via a firefly Docker image obtained from https://hub.docker.com/r/ipac/firefly/tags/"

(similar cleanup of other notebooks is needed but let's do that on another ticket)